theme/templates/pages/homepage.html

```diff
@@ -7,116 +7,290 @@
+                </div>
+                <img class="img-responsive" src="{{ STATIC_URL }}img/home-page/carousel/bg5.jpg" alt="Image 2">
+            </div>
+
+            <div class="item">
+                <div class="gray-screen"></div>
+                <div class="item-caption">
+                    <h1><strong>Collaborate</strong></h1>
+                    <h4>Upload your data or models and share them with other users. You control who has access.</h4>
+                </div>
+                <img class="img-responsive" src="{{ STATIC_URL }}img/home-page/carousel/bg2.jpg" alt="Image 3">
+            </div>
+        </div>
+
+        <ol class="carousel-controls carousel-indicators">
+            <li data-target="#home-page-carousel" data-slide-to="0" class="active">
+                <div class="carousel-control-block">
+                    <span class="carousel-control-icon glyphicon glyphicon-share "></span>
+                    <h5>SHARE</h5>
+                </div>
+            </li>
+            <li data-target="#home-page-carousel" data-slide-to="1">
+                <div class="carousel-control-block">
+                    <span class="carousel-control-icon glyphicon glyphicon-search"></span>
+                    <h5>DISCOVER</h5>
+                </div>
+            </li>
+            <li data-target="#home-page-carousel" data-slide-to="2">
+                <div class="carousel-control-block">
+                    <span class="carousel-control-icon glyphicon glyphicon-user"></span>
+                    <h5>COLLABORATE</h5>
+                </div>
+            </li>
+        </ol>
+
+        <!-- Left and right controls -->
+        <a class="left carousel-control" href="#home-page-carousel" role="button" data-slide="prev">
+            <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+        </a>
+        <a class="right carousel-control" href="#home-page-carousel" role="button" data-slide="next">
+            <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+        </a>
+    </div>
+
+    {% if user.is_anonymous %}
+    <div>
+        <div class="container">
+            <div class="row home-page-block">
+                <div class="col-xs-12">
+                    <img class="img-home-art img-responsive" src="{{ STATIC_URL }}img/logo-only.png"
+                         alt="HydroShare logo">
+                    <center><strong><h3 class="home-page-block-title">Join the community to start sharing</h3></strong>
+                    </center>
+                    <center>
+                        <span>HydroShare is an online collaboration environment for sharing data, models, and code.</span>
+                    </center>
+                    <center><a class="btn btn-primary btn-signup slideInBlock" href="sign-up/">Sign up now</a></center>
```

Comment:
Can someone help me get the sign up page to its own page and link it here to this button? I can't seem to get it to work. The problem right now is that I can't link html files outside the hs_core folder through the admin interface.